### ISSUE TITLE
aws_iam_policy does not have a description attribute

### ISSUE THREAD

**Issue by rene00** [terraform-providers/terraform-provider-aws#5808]
The aws_iam_policy resource does not have a description attribute yet it is listed in the [terraform docs](https://www.terraform.io/docs/providers/aws/r/iam_policy.html#description-1).

### Terraform Version

```
$ terraform -v
Terraform v0.11.8
+ provider.aws v1.35.0
```

### Affected Resource(s)

aws_iam_policy

### Terraform Configuration Files

```hcl
provider "aws" {
  region = "eu-west-3"
}

data "aws_iam_policy_document" "example" {
  statement {
    actions = [
      "ec2:Describe*",
    ]
    resources = ["*"]
  }
}

resource "aws_iam_policy" "example" {
  name   = "example"
  policy = "${data.aws_iam_policy_document.example.json}"
}

output "description" {
  value = "${aws_iam_policy.example.description}"
}
```

### Debug Output

```
$ terraform apply
data.aws_iam_policy_document.example: Refreshing state...
aws_iam_policy.example: Refreshing state... (ID: arn:aws:iam::REDACTED:policy/example)

Error: Error running plan: 1 error(s) occurred:

* output.description: Resource 'aws_iam_policy.example' does not have attribute 'description' for variable 'aws_iam_policy.example.description'
```

### Expected Behavior

Unsure if `description` is an AWS IAM policy attribute. If it is I would expect a default value (perhaps "") or if it is not an attribute remove this attribute from the aws_iam_policy resource.

### Actual Behavior

Error when running plan.

### Steps to Reproduce

1. `terraform apply`

**Comment by bflad** [comment]
The fix to ensure `description` is always available as an attribute (even when empty) has been merged into master and will be released with version 1.36.0 of the AWS provider, likely very shortly. 👍

**Comment by bflad** [comment]
This has been released in [version 1.36.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1360-september-13-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
